enforcer/datapath/datapath_tcp.go

```diff
@@ -622,51 +587,51 @@ func (d *Datapath) processNetworkSynAckPacket(context *pucontext.PUContext, conn
 	// Now we can process the SynAck packet with its options
 	tcpData := tcpPacket.ReadTCPData()
 	if len(tcpData) == 0 {
-		d.reportRejectedFlow(tcpPacket, nil, collector.DefaultEndPoint, context.ManagementID, context, collector.MissingToken, nil)
-		return nil, nil, errors.New("synack packet dropped because of missing token")
+		d.reportRejectedFlow(tcpPacket, nil, collector.DefaultEndPoint, context.ManagementID(), context, collector.MissingToken, nil)
+		return nil, nil, fmt.Errorf("SynAck packet dropped because of missing token")
```

Comment:
we reverted the change from errors.New to fmt.Errorf.

I researched this errors.New is better if I dont have to use any formatting and use just one fixed string. Use fmt.Errorf in case you have more to format.